internals/cli/cmd_run.go

```diff
@@ -62,22 +63,29 @@ var sharedRunEnterArgsHelp = map[string]string{
 	"--args":        "Provide additional arguments to a service",
 	"--identities":  "Seed identities from file (like update-identities --replace)",
 }
+var runArgsHelp = map[string]string{
+	"--dry": "Start {{.DisplayName}} without side-effects",
```

Comment:
It doesn't "start" Pebble though, right? I think we need a more accurate description here.